src/billing/components/PayAsYouGo/CancellationPanel.tsx

```diff
@@ -6,12 +6,37 @@ import {
   ComponentColor,
   Button,
 } from '@influxdata/clockface'
+import {track} from 'rudder-sdk-js'
 
 // Components
 import CancellationOverlay from 'src/billing/components/PayAsYouGo/CancellationOverlay'
 
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {useSelector} from 'react-redux'
+import {getQuartzMe} from 'src/me/selectors'
+import {getOrg} from 'src/organizations/selectors'
+import {event} from 'src/cloud/utils/reporting'
+
 const CancellationPanel: FC = () => {
   const [isOverlayVisible, setIsOverlayVisible] = useState(false)
+  const quartzMe = useSelector(getQuartzMe)
+  const org = useSelector(getOrg)
+
+  const handleCancelService = () => {
+    if (isFlagEnabled('trackCancellations')) {
```

Comment:
Given that Rudderstack isn't loaded into the app withoug the rudderstackReporting flag I'd think we'd want to check for that one here too.

Reply:
makes sense